src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -16,7 +16,18 @@ namespace System.Net.Http.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string Username = "testuser";
+        private const string Password = "password";
+        private const string DecompressedContentPart = "Accept-Encoding";
+        private readonly NetworkCredential _credential = new NetworkCredential(Username, Password);
 
+        private static bool JsonMessageContainsKeyValue(string message, string key, string value)
+        {
+            // Poor Man's json parsing.
```

Comment:
Actually, I don't really understand why a HTTP test is taking a dependency on Json. (I'm missing context given that the PR is missing the server-side code.)